core/src/main/java/nl/nn/adapterframework/webcontrol/api/TransactionalStorage.java

```diff
@@ -277,6 +284,48 @@ public Response resendReceiverMessages(
 		return Response.status(Response.Status.ACCEPTED).entity(errorMessages).build();
 	}
 
+	@POST
+	@RolesAllowed({"IbisDataAdmin", "IbisAdmin", "IbisTester"})
+	@Path("/adapters/{adapterName}/receivers/{receiverName}/errorstorage/changeProcessState")
```

Comment:
Now this only works for errorStorage.
If a Hold storage is introduced (which is as simple as adding a statusValueHold attribute to a JdbcTableListener), then it should be possible to move messages from errorStorage to Available and to Hold, but also from Hold to Error.
I would expect the path to be something like:
``@Path("/adapters/{adapterName}/receivers/{receiverName}/storage/{processState}/changeto/{targetState}")``